docs/core/tutorials/using-on-macos.md

```diff
@@ -129,46 +96,45 @@ namespace Library
 }
 ```
 
-This makes use of a number of modern C# features, such as 
-static usings, expression-bodied members, and interpolated strings,
-that you can learn
-about in the [Learn C#](../../csharp/index.md) section.
+The `Thing` class contains one public method, `Get`, which returns the sum of two numbers but does so by converting the sum into a string and then deserializing it into an integer. This makes use of a number of modern C# features, such as static `using` directives, expression-bodied members, and interpolated strings, that you can learn about in the [Learn C#](../../csharp/index.md) section.
 
-Now that you've updated the code, you can build the library using
-`dotnet build`.
+Build the library with the [`dotnet build`](../tools/dotnet-build.md) command. This produces a *library.dll* file under *golden/library/bin/Debug/netstandard1.4*:
 
-You now have a built *library.dll* file under *golden/library/bin/Debug/netstandard1.4*.
+```console
+dotnet build
+```
+
+## Create the test project
 
-### Writing the test project
+Build a test project for the library. From the *golden* directory, create a new test project:
 
-Let's build a test project for this library that you've build. Change to the *golden*
-directory. Run `dotnet new xunit -o test-library` to create a new test project. 
-You'll want to add this project to the solution as well by running
-`dotnet sln add test-library/test-library.csproj`.
+```console
+dotnet new xunit -o test-library
+```
 
-You'll need to add a dependency node for the library you wrote in the steps
-above. The `dotnet add reference` command does that:
+Add the test project to the solution:
 
+```console
+dotnet sln add test-library/test-library.csproj
 ```
+
+Add a dependency node for the library you created in the previous section by using the [`dotnet add reference`](../tools/dotnet-add-reference.md):
```

Comment:
"Add a dependency node" is really an implementation detail -- it might be useful if we're working directly with the file (as in the alternate step), but not here. I think I would say that we need to add a reference, and briefly explain what a reference is (the compiler otherwise has no knowledge of the library project).